recipes/metilene/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: metilene
+  version: "0.2.6"
+
+about:
+  home: http://www.bioinf.uni-leipzig.de/Software/metilene/
+  license: GPLv2
+  license_file: LICENSE
+  summary: Fast and sensitive detection of differential DNA methylation
+
+build:
+  number: 0
+
+source:
+  fn: metilene_v02-6.tar.gz
+  url: "http://www.bioinf.uni-leipzig.de/Software/metilene/metilene_v02-6.tar.gz"
+  sha256: 832255cc58d6f3d4a1a157ca6a97186f0ff66a42480d3d0e45eceb4771f3978b
+  patches:
+    - patch.0
+
+requirements:
+  build: 
+    - llvm # [osx]
+    - gcc # [linux]
+
+  run: 
+    - bedtools >=2.24
+    - perl-threaded
+    - r-ggplot2 >=2.0.0
+    - libgcc # [linux]
```

Comment:
Not everytime when you specify gcc as build dep you need libgcc as  run-dependency. So maybe you can get rid of it.